server/modules/selva/module/selva_object.c

```diff
@@ -1488,20 +1517,22 @@ int SelvaObject_GetCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int ar
     struct SelvaObject *obj;
     struct SelvaObjectKey *key;
 
-    const int ARGV_KEY = 1;
-    const int ARGV_OKEY = 2;
+    const int ARGV_LANG = 1;
```

Comment:
Actually the redis API requires that the first argument should be a key in redis. Not sure if it really matters 🤷‍♂️